flexget/components/ftp/sftp_client.py

```diff
@@ -346,10 +346,10 @@ def _connect(self, connection_tries: int) -> 'pysftp.Connection':
     def _get_cnopts(self) -> Optional['pysftp.CnOpts']:
         if not self.host_key:
             return None
-        KeyClass = getattr(
+        key_class = getattr(
             importlib.import_module("paramiko"), HOST_KEY_TYPES[self.host_key.key_type]
         )
-        key = KeyClass(data=b64decode(self.host_key.public_key))
+        key = key_class(data=b64decode(self.host_key.public_key))
```

Comment:
Not familiar with this code, but seems like this might be a class.

Reply:
KeyClass is indeed a class.  
```py
KeyClass = getattr(importlib.import_module("paramiko"), HOST_KEY_TYPES[self.host_key.key_type])
```  
The code above violates the rule '[N806](https://docs.astral.sh/ruff/rules/non-lowercase-variable-in-function/) Variable `KeyClass` in function should be lowercase'. In this case, I tend to use `# noqa` because this rule applies in most cases.